docs/UserGuide.md

```diff
@@ -342,3 +315,96 @@ Action | Format, Examples
 **Undo** | `undo`
 **Help** | `help`
 **Exit** | `exit`
+
+## List of acceptable values
+
+<style type="text/css">
+.tg  {border-collapse:collapse;border-spacing:0;}
+.tg td{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
+  overflow:hidden;padding:10px 5px;word-break:normal;}
+.tg th{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
+  font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;}
+.tg .tg-0pky{border-color:inherit;text-align:left;vertical-align:top}
+.tg .tg-fymr{border-color:inherit;font-weight:bold;text-align:left;vertical-align:top}
+</style>
```

Comment:
Should we add this at the top for better practice?

Reply:
Do you think putting these in a separate css file will be better?

Reply:
Yes I think we have a css file the UG uses, we can put it there